locations/spiders/leslies_poolmart_us.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any
+
+from scrapy.http import Response
+from scrapy.spiders import SitemapSpider
+
+from locations.hours import OpeningHours
+from locations.items import Feature
+from locations.spiders.vapestore_gb import clean_address
+from locations.structured_data_spider import extract_phone
+
+
+class LesliesPoolmartUSSpider(SitemapSpider):
+    name = "leslies_poolmart_us"
+    item_attributes = {"brand": "Leslie's Poolmart", "brand_wikidata": "Q6530568"}
+    sitemap_urls = ["https://lesliespool.com/robots.txt"]
+    sitemap_rules = [("/location/", "parse")]
+
+    def parse(self, response: Response, **kwargs: Any) -> Any:
+        item = Feature()
+        item["ref"] = response.xpath("////a[@data-store-id]/@data-store-id").get()
+        item["lat"] = response.xpath("//@data-store-latitude").get()
+        item["lon"] = response.xpath("//@data-store-longitude").get()
```

Comment:
All the lat/lons seem to be set to the same point in the map. Is this pulling the wrong location somehow?